### PR TITLE
(enh) moonscript supports eval now

### PR DESCRIPTION
**Pull request by joshgoebel**

### CODE REVIEW

File: src/api/moonscript.c [hunk 1]
```diff
@@ -91,6 +108,9 @@ static bool initMoonscript(tic_mem* tic, const char* code)
             return false;
         }
 
+        lua_setglobal(lua, "_ms_loadstring");
+        lua_getglobal(lua, "_ms_loadstring");
```
Comment (joshgoebel) [inline src/api/moonscript.c:112]: Dunno if there is a better way to do this... I was looking for `lua_tee` or something, but couldn't find.

Comment (nesbox) [reply src/api/moonscript.c:112]: Honestly, I don't understand what's going on here :)
Also, could you pls move the `"_ms_loadstring"` to `static const char _ms_loadstring[] = "_ms_loadstring"` variable?

Comment (joshgoebel) [reply src/api/moonscript.c:112]: Well, Lua is stack based, so setting a global removes it from the stack so you have to put it back on the stack... I was looking for a "duplication item on top of stack" API, but didn't see it.

**Review by nesbox — APPROVED** [review]
Thank you

**Pull request merged**